save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -18,16 +18,18 @@ class TestExecutionService(private val testResultRepository: TestExecutionReposi
     fun saveTestResult(testExecutions: List<TestExecution>) {
         testExecutions.forEach { testExec ->
             val foundTestExec = testResultRepository.findById(testExec.id)
-            if (!foundTestExec.isPresent) {
-                log.error("Test execution with ${testExec.id} id was not found")
-            } else {
-                foundTestExec.get().run {
-                    this.startTime = testExec.startTime
-                    this.endTime = testExec.endTime
-                    this.status = testExec.status
-                    testResultRepository.save(this)
-                }
-            }
+            foundTestExec.ifPresentOrElse(
+                {
```

Comment:
This lambda is  a consumer for optional value, no need to call `get` inside of it